frontend/src/components/SymmetricCountryChart.js

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+import ChartSection from './Chartsection'
+
+export default function SymmetricCountryChart ({countries}) {
+    return(<Wrapper>
+        <ChartSection displayedCountries={countries} entry={'total'} heading={'Total Score'}/>
+        <ChartSection displayedCountries={countries} entry={'freedom'} heading={'Democracy and Freedom'}/>
+        <ChartSection displayedCountries={countries} entry={'environment'} heading={'Environment and Climate'}/>
+        <ChartSection displayedCountries={countries} entry={'gender'} heading={'Gender Equality'}/>
+        <ChartSection displayedCountries={countries} entry={'lgbtq'} heading={'LGBTQ Acceptance'}/>
+        <ChartSection displayedCountries={countries} entry={'inequality'} heading={'Income Equality'}/>
+        <ChartSection displayedCountries={countries} entry={'corruption'} heading={'Corruption Control'}/>
+    </Wrapper>)
+}
+
+SymmetricCountryChart.propTypes = {
+    countries: PropTypes.object
+}
+
+const Wrapper = styled.section`
+    padding-top: 20px;
+    text-align: center;
+    width: 375px;
```

Comment:
Be a bit aware of using pixel values, it makes the screen less responsive. So here check if you can do the padding-top and width with more responsive options than px. :)